addons/payment_paypal/models/payment_transaction.py

```diff
@@ -69,11 +55,32 @@ def _get_specific_processing_values(self, processing_values):
                         'admin_area_1': self.partner_state_id.name,
                         'admin_area_2': self.partner_city,
                         'postal_code': self.partner_zip,
-                        'country_code': self.partner_country_id.code,
+                        'country_code': self.partner_country_id.code or self.company_id.country_id.code,
```

Comment:
Extract in a separate variable before `payload`.
```suggestion
                        'country_code': country_code,
```